sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -142,27 +142,29 @@ class AstBuilder extends DataTypeAstBuilder with SQLConfHelper with Logging {
     CompoundBody(buff.toSeq, label)
   }
 
-  override def visitBeginEndCompoundBlock(ctx: BeginEndCompoundBlockContext): CompoundBody = {
-    val beginLabelCtx = Option(ctx.beginLabel())
-    val endLabelCtx = Option(ctx.endLabel())
+  override def visitBeginEndCompoundBlock(ctx: BeginEndCompoundBlockContext): CompoundBody =
+    withOrigin(ctx) {
```

Comment:
we are not using origin anywhere in this function, maybe remove for now and we can add if needed later.